src/core/core.pro

```diff
@@ -64,7 +64,10 @@ HEADERS += \
     recentprojectlistmodel.h \
     referencingfeaturelistmodel.h \
     featurechecklistmodel.h \
-    qfieldappauthrequesthandler.h
+    qfieldappauthrequesthandler.h \
+    fileutils.h \
+    androidviewstatus.h \
```

Comment:
If you want to keep this in a separate file it has to go to the android only section in .pro (and CMakeFiles)